app/controllers/agents/creneaux/agent_searches_controller.rb

```diff
@@ -3,6 +3,7 @@ class Agents::Creneaux::AgentSearchesController < AgentAuthController
 
   def index
     skip_policy_scope
+    @user = params[:user]
```

Comment:
we try to suffix the variable names with `_id` when they contain IDs and not full blown AR records. I don't think it's possible to pass a AR record through params, so I believe here it should be `@user_id = params[:user_id]` or even better `@user = User.find(params[:user_id]) if params[:user_id].present?`